calculatorV2.rb

```diff
@@ -0,0 +1,120 @@
+# Calculator program
+
+# Get operation input from user
+puts "Hey-oh! What kind of calculation would you like to perform?"
+operation = gets.chomp
+
+until operation == "^" || operation == "exp" || operation == "exponent" || operation == "%" || operation == "mod" || operation == "modulo" || operation == "*" || operation == "mult" || operation == "multiply" || operation == "/" || operation == "div" || operation == "divide" || operation == "-" || operation == "sub" || operation == "subtract" || operation == "+" || operation == "add" || operation == "addition"
+	puts "I\'m sorry, I don't understand. Please try again"
+	#kick it back to retrieve new operation and reassign operation
+	operation = gets.chomp
+end 	
+
+
+
+
+# Get first number input from user
+puts "Awesome. What is the first number you would like to perform that on?"
+first_number = gets.chomp
+
+if first_number == "0"
+	first_number_int = first_number.to_i
+
```

Comment:
I like how clear your variables are here. One idea to shorten up the long one might be first_int. Since ints are numbers and data types, you don't really lose any information.